app/src/containers/HomePage/HomePage.js

```diff
@@ -36,6 +28,7 @@ class HomePage extends Component {
     if (this.props.isAuthenticated) {
       let fbID = this.props.profile.user_id
       let id = fbID.split('|')
+      console.log('id', id)
```

Comment:
Let's utilize the forthcoming refactor PR to also factor out all references to console.log